telos-frontend/src/components/steam/WidgetSteam.jsx

```diff
@@ -0,0 +1,10 @@
+import SportsEsportsIcon from '@material-ui/icons/SportsEsports';
+import styles from '../Widget.module.css';
+
+const WidgetClock = ({ addNewSteam }) => (
+  <div onClick={addNewSteam} className={`${styles.widgetIcon} ${styles.vertical}`}>
+    <SportsEsportsIcon style={{ fontSize: 40 }} />
+  </div>
+);
+
+export default WidgetClock;
```

Comment:
Same here ^

Reply:
Another Nice spotting! I have resolved the issue.